package.json

```
@@ -5,7 +5,16 @@
   "scripts": {
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
-    "lint": "vue-cli-service lint"
+    "format": "prettier --write src/main.js",
+    "lint": "eslint --fix src/main.js",
+    "fix": "npm run format && npm run lint",
+    "prepare": "husky install"
+  },
+  "lint-staged": {
+    "*.js": [
+      "prettier --write",
+      "eslint --fix"
+    ]
```

husky入れたのナイスだね！！

ただフォーマットのタイミングなんだけどコミット前に勝手に整形されちゃうと意図したコードと違うものがコミットされちゃう（意図しないフォーマットになる場合がある）から個人的にはエディタの機能使ってセーブした時にフォーマットされるようにしてコミット前にちゃんと自分のコード確認してからコミットするって方がいいと思ってる。
もしそれに賛同してもらえるならここではチェックだけにしたほうがいいかな。もし時間がかかりそうなら一旦今のままでOK！

ちゃんと動くかはわからないけどイメージはこんな感じ
```suggestion
  "lint-staged": {
    "*.js": [
      "prettier --check",
    ]
```
https://prettier.io/docs/en/cli.html#--check

---

> コミット前に勝手に整形されちゃうと意図したコードと違うものがコミットされちゃう
これに関しては認知していて、PRにもありますがセーブ時にしようかはは結構悩みました。。

とはいえ、そもそも今時点ではそこまで複雑なコードは出てこないであろうという面と、私もかずきさんもフォーマッターよりここはこうした方がいいとなることはかなり少ないことが予想されるので、一旦これで運用してみたい（ぶっちゃけ私の興味もある）感じです 笑